imagemodal/imagemodal.py

```diff
@@ -1,76 +1,35 @@
 """
 This is the core logic for the Image Modal XBlock
 """
-import os
-
-import pkg_resources
-
 from django.utils.translation import ugettext_lazy as _
-
 from xblock.core import XBlock
 from xblock.fields import Scope
 from xblock.fields import String
-from xblock.fragment import Fragment
-
+from xblockutils.resources import ResourceLoader
 from xblockutils.studio_editable import StudioEditableXBlockMixin
 
-DEFAULT_FIELDS = [
-    'parent',
-    'tags',
-]
+from .mixins.fragment import XBlockFragmentBuilderMixin
+from .mixins.scenario import ImageModalScenarioMixin
 
-
-def get_resource_string(path):
-    """
-    Retrieve string contents for the file path
-    """
-    path = os.path.join('public', path)
-    resource_string = pkg_resources.resource_string(__name__, path)
-    return resource_string.decode('utf8')
+URL_FONT_AWESOME_CSS = '/'.join([
```

Comment:
This is a little ugly, to overcome long lines.
There's a more holistic fix in a follow-up PR.